Add reconstruction notebooks

RolfAdolf opened the pull request:
Добавил ноутбуки с реализацией реконструкций.

RolfAdolf committed:
Add reconstruction notebooks

RolfAdolf committed:
Add python code

RolfAdolf committed:
Change loftr files.

nataliakosharich commented:
Fix all commits messages according to this rules https://cbea.ms/git-commit/

nataliakosharich commented:
Check this article https://mydeveloperplanet.com/2018/10/31/git-lfs-why-and-how-to-use/. Add large and bin files like jpg to gitlfs

nataliakosharich commented on loftr_Code.py:
```diff
@@ -0,0 +1,58 @@
+# Необходимые модули
```
Use only english in source code

nataliakosharich replied:
Do not write obvious comments. Check "Clean code" summary here https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29

nataliakosharich commented on loftr_Code.py:
```diff
@@ -0,0 +1,58 @@
+references = [p.relative_to(images).as_posix() for p in (images).iterdir()]
+
+# Вывод
+print(len(references), "mapping images")
```
Use python string formatting like this https://realpython.com/python-string-formatting/#3-string-interpolation-f-strings-python-36

nataliakosharich commented on environment.yml:
```diff
@@ -0,0 +1,260 @@
+  - zstd=1.5.2=h6239696_4
+  - pip:
+    - pycolmap==0.3.0
+prefix: /home/nadir/anaconda3/envs/StudProject
```
Do not commit local setting files (add to ignore). This file contains your local path and this path doesn't have any sense for another developer. Modify it to exclude path or exclude it from repo.

nataliakosharich commented on Notebooks/README.md:
```diff
@@ -0,0 +1,29 @@
+# LoFTR
```
Translate to english

nataliakosharich commented:
Pr should be opened on main branch

RolfAdolf committed:
Correct all remarks

All marks were taken into account:

- Commits are okay (?)

- Git LFS

- Only useful comments left

- Everything was translated into English

- String formatting

- .gitignore file

RolfAdolf committed:
Delete environment
